src/modules/core/sagas/utils/getGasPrices.ts

```diff
@@ -97,18 +97,33 @@ const fetchGasPrices = async (
       // API prices are in Gwei, so they need to be normalised
       const oneGwei = bigNumberify(10 ** 9);
 
+      let { average } = data;
+      let { fast } = data;
+      /*
+       * @NOTE IF we're on Gnosis Chain, ensure that transactions alway pay at
+       * least 2gwei for gas
+       *
+       * This to counteract some unpleasantness coming from the Blockscout gas oracle
+       *
+       * Locally this is not a problem (and we don't even use the oracle to estimate)
+       */
+      if (
+        DEFAULT_NETWORK === Network.Xdai ||
+        DEFAULT_NETWORK === Network.XdaiFork
+      ) {
+        average = data.average > 2 ? data.average : 2;
+        fast = data.fast > 2 ? data.fast : 2;
```

Comment:
Could be: 

``` js
average = average > 2 ? average : 2;
fast = fast > 2 ? fast : 2;
```

Reply:
I thought about this change, but I wanted to emphasize that the values are coming from the oracle _(`data`)_.

So I'll leave this as it is.